### ISSUE TITLE
is gitattributes file still used/useful

### ISSUE THREAD

**Issue by jspaaks** [research-software-directory/research-software-directory#249]
https://github.com/research-software-directory/research-software-directory/blob/3dc67b5654aab912b6503a403c02b84a43d900bc/.gitattributes

**Comment by Tommos0** [comment]
I don't think so anymore! You're storing all secrets outside of the Git repo now right?
